src/Spaces/pointspace.jl

```diff
@@ -5,15 +5,15 @@ local_geometry_data(space::AbstractPointSpace) = space.local_geometry
 """
     PointSpace <: AbstractSpace
 
-A zero-dimensional space. 
+A zero-dimensional space.
 """
 struct PointSpace{LG} <: AbstractPointSpace
     local_geometry::LG
 end
 
 function PointSpace(local_geometry::LG) where {LG <: Geometry.LocalGeometry}
     FT = Geometry.undertype(LG)
-    local_geometry_data = DataLayouts.DataF{LG}(Matrix{FT})
+    local_geometry_data = DataLayouts.DataF{LG}(Array{FT})
```

Comment:
Shouldn't we be specifying the size? What will this do exactly?

Reply:
Ever since fixing inference issues in Thermodynamics, I've tried avoiding UnionAll types

Reply:
just determines what type (Array, CuArray) and floating point type (Floa32, Float64). Dimensionality is handled by the constructor